### PR TITLE
fix: cannot emit `load` event in react wrapper

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* related issue(#1808)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다.

File: apps/react-editor/src/viewer.tsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-// @ts-ignore
-import Viewer from '@toast-ui/editor/dist/toastui-editor-viewer';
+import Viewer, { EventMap } from '@toast-ui/editor/dist/toastui-editor-viewer';
```
Comment (adhrinae) [inline apps/react-editor/src/viewer.tsx:2]: 그냥 궁금한건데, `Viewer` 는 `dist` 폴더에서 가져와야 하는 이유가 있나요?

**Review by adhrinae — APPROVED** [review]
리뷰 완료합니다.

File: apps/react-editor/src/viewer.tsx (hunk 1 above)
Comment (js87zz) [reply apps/react-editor/src/viewer.tsx:2]: 어플리케이션의 기본 export에서는 에디터만 제공합니다~! 뷰어는 번들 용량 문제도 있어서 별도 dist로 제공하고 있습니다.

**Review by jwlee1108 — COMMENTED** [review]
리뷰 완료합니다.

**Pull request merged**